cypress/e2e/pages/mediaAssetPage/index.cy.js

```diff
@@ -1,11 +1,353 @@
+    tests: canonicalTests,
+  },
+  {
+    path: '/zhongwen/simp/world-53107744', // CPS video
+    service: 'zhongwen',
+    runforEnv: ['live'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/zhongwen/simp/uk-23283128', // CPS Audio
+    service: 'zhongwen',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/zhongwen/simp/multimedia/2016/11/161107_tc2_testmap1?renderer_env=test', // TC2 Video
+    service: 'zhongwen',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/zhongwen/trad/world-53107744', // CPS video
+    service: 'zhongwen',
+    runforEnv: ['live'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/zhongwen/trad/uk-23283128', // CPS Audio
+    service: 'zhongwen',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+  {
+    path: '/zhongwen/trad/multimedia/2016/11/161107_tc2_testmap1?renderer_env=test', // TC2 Video
+    service: 'zhongwen',
+    runforEnv: ['test'],
+    tests: canonicalTests,
+  },
+];
+
+// non Amp Non Smoke Test Suites
+const nonAmpOnlyNonSmokeTestSuites = Cypress.env('SMOKE')
+  ? []
+  : [
+      {
+        path: '/afrique/institutionelles/2015/07/150714_hissene_habre_explainer', // TC2 MAP
+        service: 'afrique',
+        runforEnv: ['live'],
+        tests: canonicalTests,
+      },
+      {
+        path: '/arabic/multimedia/2016/06/160601_qatar_sewika_smoking', // TC2 video
+        service: 'arabic',
+        runforEnv: ['live'],
+        tests: canonicalTests,
+      },
+      {
+        path: '/hindi/multimedia/2015/09/150921_what_is_innovation_ms', // TC2 video
+        service: 'hindi',
+        runforEnv: ['live'],
+        tests: canonicalTests,
+      },
+      {
+        path: '/russian/multimedia/2012/04/120411_v_titanic_last_survivor', // TC2 video
+        service: 'russian',
+        runforEnv: ['live'],
+        tests: canonicalTests,
+      },
+      {
+        path: '/swahili/medianuai/2016/05/160517_apatae_fatacky', // TC2 MAP with video clip
+        service: 'swahili',
+        runforEnv: ['live'],
+        tests: canonicalTests,
+      },
+      {
+        path: '/urdu/multimedia/2014/11/141104_hindu_riaz_kq', // TC2 MAP with video clip
+        service: 'urdu',
+        runforEnv: ['live'],
+        tests: canonicalTests,
+      },
+      {
+        path: '/zhongwen/simp/multimedia/2016/06/160608_vid_gaokao_voxpop', // TC2 video
+        service: 'zhongwen',
+        runforEnv: ['live'],
+        tests: canonicalTests,
+      },
+      {
+        path: '/zhongwen/trad/multimedia/2016/06/160608_vid_gaokao_voxpop', // TC2 video
+        service: 'zhongwen',
+        runforEnv: ['live'],
+        tests: canonicalTests,
+      },
+    ];
 
-runTestsForPage(testsForPage);
+const canonicalTestSuites = Cypress.env('SMOKE')
+  ? canonicalSmokeTestSuites
+  : canonicalNonSmokeTestSuites;
+
+const ampTestSuites = canonicalTestSuites.map(testSuite => {
+  return {
+    ...testSuite,
+    path: `${testSuite.path}.amp`,
```

Comment:
This won't work for any of the paths which have renderer_env=test at the end - you'll need to use ``getPathWithSuffix` e.g. https://github.com/bbc/simorgh/blob/latest/cypress/e2e/specialFeatures/atiAnalytics/index.cy.js#L476